src/rules/maxStatementsWithExclusions/mockCallback.js

```diff
@@ -0,0 +1,15 @@
+const lodash = require('lodash');
+
+const mockCallback = (node) => {
+  try {
+    const test = node.body.map((n) => {
+      return lodash.get(n, 'expression.callee.object.name', undefined);
+    });
+    const filtered = test.filter((t) => t !== 'logger').length;
+    return filtered;
```

Comment:
Useless var

Reply:
this is done (in the file that's in the new location)